tests/test_error_handling_integration.py

```
@@ -345,22 +350,23 @@ def test_state_machine_maintains_last_known_good_state(self):
         # Should still be in initial phase
         assert sm.get_current_phase() == initial_phase
     
-    def test_anomaly_detector_recovers_after_error(self):
+    @pytest.mark.asyncio
+    async def test_anomaly_detector_recovers_after_error(self):
         """Test anomaly detector can recover from errors."""
         # First call with valid data
         data1 = {"voltage": 8.0, "temperature": 25.0, "gyro": 0.01}
-        is_anomalous1, score1 = detect_anomaly(data1)
+        is_anomalous1, score1 = await detect_anomaly(data1)
```

Ruff (0.14.10)

Unpacked variable `score1` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)